### PR TITLE
fix preview for models with materialized == ephemeral

### PR DESCRIPTION
**Pull request by pgrivachev**
For some weird reasons dbt-rpc doesn't compile models with materialized == ephemeral. I added workaround.

### CODE REVIEW

File: server/src/ModelCompiler.ts [hunk 1]
```diff
@@ -71,7 +74,13 @@ export class ModelCompiler {
           if (result.isErr()) {
             this.onCompilationErrorEmitter.fire(result.error);
           } else {
-            this.onCompilationFinishedEmitter.fire(result.value);
+            const value =
+              // For some reason rpc server don't compile intermediate models for packages, for this case we get results from target folder
+              result.value === ' ' && modelPath.includes('intermediate') && !modelPath.includes(path.sep)
```
Comment (fivetran-dmitryboykov) [inline server/src/ModelCompiler.ts:79]: Seems that the problem is not in `intermediate`. Please, see installed packages configs:
**dbt-language-server/e2e/projects/project-with-packages:**
```
packages:
  - package: fivetran/github
    version: 0.4.0
```
_dbt_packages/github/dbt_project.yml:_
```
models:
  github:
    materialized: table
    intermediate:
      materialized: ephemeral
```

**My test project:**
```
packages:
  - package: fivetran/salesforce
    version: 0.5.1
```
_dbt_packages/salesforce/dbt_project.yml_
```
models:
  salesforce:
    materialized: table
    intermediate:
      materialized: ephemeral
```

Comment (fivetran-dmitryboykov) [reply server/src/ModelCompiler.ts:79]: I don't know how different versions of rpc server will handle `ephemeral` models. Please consider obtaining compiled sql for all packages from files in `target`.

**Review by fivetran-dmitryboykov — CHANGES_REQUESTED** [review]
Looks good. Please review my note.

File: server/src/ModelCompiler.ts (hunk 1 above)
Comment (pgrivachev) [reply server/src/ModelCompiler.ts:79]: Thanks for the great finds, I'll try to figure it out.

Comment (pgrivachev) [reply server/src/ModelCompiler.ts]: Added fallback for all models with materialized == ephemeral.

**Review by fivetran-dmitryboykov — APPROVED** [review]

**Pull request merged**